source/_components/sensor.waze_travel_time.markdown

```diff
@@ -0,0 +1,46 @@
+---
+layout: page
+title: "Waze Travel Time"
+description: "Instructions on how to add Waze travel time to Home Assistant."
+date: 2018-01-23 00:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: waze.png
+ha_category: Transport
+ha_iot_class: "Cloud Polling"
+ha_release: 0.61
+---
+
+Sensor to provide travel time from the [WazeRouteCalculator](https://github.com/kovacsbalu/WazeRouteCalculator).
+
+The sensor will update the travel time every 5 minutes by default.
+Unit system is set to metric system (at least for the moment).
+
+```yaml
+# Example entry for configuration.yaml
+sensor:
+  - platform: waze_travel_time
+    name: "Example Name"
+    origin: Montréal, QC
+    destination: Québec, QC
+    region: 'US'
+    update_interval: '00:03'
+    outputs:
+     - duration
+     - distance
+     - route
+```
+
+Configuration variables:
```

Comment:
Please use the configuration tags, for more information please see:
https://home-assistant.io/developers/documentation/create_page/#configuration

Reply:
I've made the requested changes using the new tags